README.md

```diff
@@ -1,65 +1,50 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Hello Microverse!
 
-> One paragraph statement about the project.
+> A simple project setup  with html/css with support for linters and web accessibility.
 
-![screenshot](./app_screenshot.png)
-
-Additional description about the project and its features.
+![screenshot](./screenshot.png)
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
+- Stylelint
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link]()
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
+- NPM
+- Web browser
 
 ### Setup
-
-### Install
+- Clone the repo to your local machine.
```

Comment:
- [ ] I think it's best to be more descriptive here. Add some more instructions on how to clone the project like `git clone repo_link` so this way it will be more clearer to any user.